spec/support/capybara_poltergeist.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.configure do |config|
+  Capybara.default_max_wait_time = 60 # Seconds to wait before timeout error. Default is 2
```

Comment:
60 seconds is too much. When it waits for an error it will take those 60 seconds. I would recommend 5 max.